<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLogicGates">
       <attribute name="title">
        <string>Logic Gates</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="btnGateBuffer">
          <property name="toolTip">
           <string>Buffer Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/BUFFER.png</normaloff>:/Icons/Resources/BUFFER.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateNot">
          <property name="toolTip">
           <string>Not Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/NOT.png</normaloff>:/Icons/Resources/NOT.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateAnd">
          <property name="toolTip">
           <string>And Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/AND.png</normaloff>:/Icons/Resources/AND.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateOr">
          <property name="toolTip">
           <string>Or Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/OR.png</normaloff>:/Icons/Resources/OR.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateNand">
          <property name="toolTip">
           <string>Nand Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/NAND.png</normaloff>:/Icons/Resources/NAND.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateNor">
          <property name="toolTip">
           <string>Nor Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/NOR.png</normaloff>:/Icons/Resources/NOR.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateXor">
          <property name="toolTip">
           <string>Xor Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/XOR.png</normaloff>:/Icons/Resources/XOR.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnGateXnor">
          <property name="toolTip">
           <string>Xnor Gate</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Resources/XNOR.png</normaloff>:/Icons/Resources/XNOR.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabInputOutput">
       <attribute name="title">
        <string>Input/Output</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="btnIOLowConstant">
          <property name="toolTip">
           <string>Low Constant</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnIOHighConstant">
          <property name="toolTip">
           <string>High Constant</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnIOToggleButton">
          <property name="toolTip">
           <string>Toggle Button</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnIOLightBulb">
          <property name="toolTip">
           <string>Light Bulb</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabIntegratedCircuits">
       <attribute name="title">
        <string>Integrated Circuits</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QScrollArea" name="tabICScrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tabICScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>844</width>
             <height>64</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="tabICInfoLabel">
              <property name="text">
               <string>This tab allows you to import .csim files from the directory the file is saved in. To use integrated circuits, please save or open another file.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelZoomLevel">
         <property name="text">
          <string>100%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnZoomIn">
         <property name="toolTip">
          <string>Zoom In</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="zoom-in">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="zoomSlider">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnZoomOut">
         <property name="toolTip">
          <string>Zoom Out</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="zoom-out">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionRemove_Connections"/>
    <addaction name="actionSelect"/>
    <addaction name="actionPan"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Undo_View"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionStep"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTutorial"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarTop">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionReset_Zoom"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionRemove_Connections"/>
   <addaction name="actionSelect"/>
   <addaction name="actionPan"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionStep"/>
  </widget>
  <widget class="QToolBar" name="toolBarBottom">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="draw-connector">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Connect Parts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-select">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="icon">
    <iconset theme="zoom">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="hand">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRemove_Connections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="format-remove-node">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove Connections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset theme="debug-step-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Step</string>
   </property>
  </action>
  <action name="actionToggle_Undo_View">
   <property name="icon">
    <iconset theme="edit-undo-history">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Toggle Undo View</string>
   </property>
   <property name="toolTip">
    <string>Toggle Undo View</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="icon">
    <iconset theme="help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
